README.md

```diff
@@ -1,6 +1,6 @@
-# Jupyter keymaps for VS Code
+# Jupyter Keymaps Extension for Visual Studio Code
 
-This extension provides Jupyter-style keymaps for VS Code notebooks.
+This extension provides Jupyter-consistent keymaps for notebooks in Visual Studio Code. This extension comes with the [Jupyter extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) and can be disabled or uninstalled.
```

Comment:
Jupyter-consistent seems like a funny hyphenation. I would say "This extension provides keymaps for notebooks in Visual Studio Code to match the hotkeys used in Jupyter" or something more like that. But just a matter of taste so feel free to ignore.

Reply:
Maybe a bit obvious, but might be worth saying the why for "disabled or uninstalled" like "and can be disabled or uninstalled separately to use different keymaps for notebooks."